New version: SpheriCart v0.1.1

JuliaRegistrator opened the pull request:
- Registering package: SpheriCart
- Repository: https://github.com/lab-cosmo/sphericart
- Created by: @cortner
- Version: v0.1.1
- Commit: 309b614189c096f035707c21f0425f5ef915599b
- Reviewed by: @cortner
- Reference: https://github.com/lab-cosmo/sphericart/issues/86#issuecomment-2179662942
- Description: Multi-language library for the calculation of spherical harmonics in Cartesian coordinates
<!-- daf33ad5e6d085e55119bc06703763cec7c0b68dd6e421de144489cbdea594521fe902435af960eb7412f27e74e019e410300bd3a2637f95872d8e0b2e20b9b0b2b6d7259133f1dd731a05f62f847893f5260ed7462d22e984fee34ff6972089c0ac136ec3cb15c444d957b0ddf282b7bdd8b25ada73dd13e4df0afc2093c5158b67c2d998f8036c33c34584feb876db0c57ebe3d7ea690d6cbfe1401f5b14d44dac528944a5f77041da189bdaa02a01 -->

cortner commented:
There is a `LICENSE.md` in the top folder of the package and a `LICENSE.md` in the sub-folder where there julia code lives. Can somebody explain to me please why this is not automerged? 

To be clear I am not interested in manually merging this PR. I need to the automerge functionality for the future.

JuliaRegistrator committed:
New version: SpheriCart v0.1.1

UUID: 5caf2b29-02d9-47a3-9434-5931c85ba645
Repo: https://github.com/lab-cosmo/sphericart.git
Tree: cec8f2ccc393a52abe874253c91b79f8808a65c2

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
